app/[lang]/onboarding/page.tsx

```
@@ -8,7 +8,11 @@ const Onboarding = async ({
 }) => {
   const dict = await getDictionary(lang as Lang);
```

Could you check if `lang` actually is of type `Lang` before casting it? And then maybe return an error (404 maybe?) if it is not